### DISCUSSION TITLE
Adding a prefix to fonts loaded in css file.

### DISCUSSION THREAD

**Discussion by Brammm** [symfony/symfony#42720] [Q&A]
I’m running a symfony app with a couple React components behind a reverse proxy on a subdirectory (the app is accessed through example.com/v2/prefix/ which is then proxied to a separate container). I’ve managed to set everything up as needed and my css and js all loads fine EXCEPT for my fonts.

The font in my CSS is loaded through
```
src: url(~../../public/assets/fonts/AlternateGothicNo3-d.ttf);
```
but in the css file eventually, that becomes
```
src: url(/build/fonts/AlternateGothicNo3-d.414b70e7.ttf);
```
while it should be
```
src: url(/v2/prefix/build/fonts/AlternateGothicNo3-d.414b70e7.ttf);
```
Is there some way to fix this? (edited)

**Comment by chyshyya** [comment]
This is not symfony problem but package manager you are using to build final css like Webpack.
If so look for `resolve-url-loader` module for Webpack to add prefix to your final url(prefix/fonts/... and in webpack config set loader for .ttf as:
`sass-loader: 'css-loader', 'resolve-url-loader', 'sass-loader'`
[https://github.com/bholloway/resolve-url-loader/blob/v5/packages/resolve-url-loader/README.md](https://github.com/bholloway/resolve-url-loader/blob/v5/packages/resolve-url-loader/README.md)
Look for `root` option.
